pkg/controller/awsfederatedrole/awsfederatedrole_controller.go

```diff
@@ -87,9 +87,23 @@ type ReconcileAWSFederatedRole struct {
 func (r *ReconcileAWSFederatedRole) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Controller", controllerName, "Request.Namespace", request.Namespace, "Request.Name", request.Name)
 
+	configMap, err := utils.GetOperatorConfigMap(r.client)
+	if err != nil {
+		log.Error(err, "failed retrieving configmap")
+	}
+	ifFedramp, err := utils.IsFedramp(configMap)
+	if err != nil {
+		log.Error(err, "Unable to verify if cluster is fedramp")
+	}
```

Comment:
Wouldn't we want to return an error if we can't read the config map and cancel reconcilation?
I see these lines duplicated multiple times in the PR, probably worth encapsulating in their own function?